go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/soypat/cyw43439
 
-go 1.20
+go 1.23.7
```

Comment:
If possible we'd want version to remain at 1.20

Reply:
Reverted to 1.20